tests/test_multioutput.py

```diff
@@ -291,8 +291,11 @@ def test_sample_conditional_mixedkernel():
     np.testing.assert_array_almost_equal(np.cov(value, rowvar=False), np.cov(value2, rowvar=False), decimal=1)
 
 
-@pytest.mark.parametrize('R', [1, 5])
-@pytest.mark.parametrize("func", [fully_correlated_conditional_repeat, fully_correlated_conditional])
```

Comment:
The case with R=5 and func=fully_correlated_conditional is not valid as fully_correlated_conditional requires R=1

Reply:
how come it was working before?

Reply:
because it simply did `[0]` instead of `tf.squeeze()`.